src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        return Color.values()[new Random().nextInt(Color.values().length)].toString().toLowerCase();
```

Comment:
Lets make Random class level parameter (not local), than we won't create new instance of Random class every time when we call getRandomColor() method